diff --git a/node_modules/ember-bootstrap/addon/components/bs-modal.js b/node_modules/ember-bootstrap/addon/components/bs-modal.js
index 0e42794..021599e 100644
--- a/node_modules/ember-bootstrap/addon/components/bs-modal.js
+++ b/node_modules/ember-bootstrap/addon/components/bs-modal.js
@@ -522,7 +522,7 @@ export default class Modal extends Component {
     await nextRunloop();
 
     const { backdropElement } = this;
-    assert('Backdrop element should be in DOM', backdropElement);
+   // assert('Backdrop element should be in DOM', backdropElement);
 
     await transitionEnd(backdropElement, this.backdropTransitionDuration);
   }
@@ -541,7 +541,7 @@ export default class Modal extends Component {
 
     if (this.usesTransition) {
       const { backdropElement } = this;
-      assert('Backdrop element should be in DOM', backdropElement);
+  //    assert('Backdrop element should be in DOM', backdropElement);
 
       await transitionEnd(backdropElement, this.backdropTransitionDuration);
     }
