diff --git a/node_modules/ember-auto-import/js/webpack.js b/node_modules/ember-auto-import/js/webpack.js
index 4ccbe91..f294b4b 100644
--- a/node_modules/ember-auto-import/js/webpack.js
+++ b/node_modules/ember-auto-import/js/webpack.js
@@ -201,6 +201,7 @@ class WebpackBundler extends broccoli_plugin_1.default {
                     this.babelRule(stagingDir, (filename) => this.fileIsInApp(filename), this.opts.rootPackage.babelOptions),
                     {
                         test: /\.css$/i,
+                        resourceQuery: /^((?!raw).)*$/i,
                         use: [
                             styleLoader,
                             {
@@ -209,6 +210,11 @@ class WebpackBundler extends broccoli_plugin_1.default {
                             },
                         ],
                     },
+                    {
+                       test: /\.css$/i,
+                       resourceQuery: /raw/i,
+                       type: 'asset/source',
+                    }
                 ],
             },
             node: false,
